core/src/main/scala/stryker4s/extension/mutationtype/MutationTypes.scala

```diff
@@ -18,7 +18,8 @@ object Mutation {
     classOf[BooleanLiteral].getSimpleName,
     classOf[LogicalOperator].getSimpleName,
     classOf[StringLiteral[_]].getSimpleName,
-    classOf[MethodExpression].getSimpleName
+    classOf[MethodExpression].getSimpleName,
+    classOf[PatternMatchExpression].getSimpleName
```

Comment:
I don't think we should add an extra category. That means we won't follow the [category standard](https://github.com/stryker-mutator/stryker-handbook/blob/master/mutator-types.md#supported-mutators) anymore. I am not sure which category it should be in instead. Perhaps block expression

Reply:
I renamed it to `RemoveStatement` for now. Should probably discuss further what we want to call this.